src/main/webapp/WEB-INF/pages/messages.ftl

```diff
@@ -0,0 +1,137 @@
+<#include "commons.ftl">
+
+<section class="content-header">
+      <h1><@spring.message 'MESSAGES.PAGE.TITLE'/></h1>
+      <ol class="breadcrumb">
+          <li><a href="/market-time/dashboard"><i class="fa fa-dashboard"></i><@spring.message 'NAVIGATION.HOME'/></a></li>
+          <li class="active"><@spring.message 'NAVIGATION.MESSAGES.PAGE'/></li>
+      </ol>
+</section>
+
+<section class="content">
+    <div class="box box-success">
+        <div class="box-header with-border"></div>
+        <div class="row-style row">
+            <div class="padding-5 col-xs-12 col-sm-offset-1 col-sm-10">
+                <div class="inbox-head">
+                    <form action="#" class="pull-right position">
+                        <div class="input-append">
+                            <input type="text" class="search-input" placeholder="<@spring.message 'MESSAGES.SEARCH.PLACEHOLDER'/>">
+                            <button class="btn search-btn" type="button"><i class="fa fa-search"></i></button>
+                        </div>
+                    </form>
+                </div>            
+                <div class="col-sm-12 container">
+                    <div class="tab-content">
+                        <ul class="nav nav-tabs">
+                            <li id="inbox-tab" class="active"><a data-toggle="tab" href="#inbox"><@spring.message 'MESSAGES.INBOX.TAB'/></a></li>
+                            <li id="compose-tab"><a data-toggle="tab" href="#outbox"><@spring.message 'MESSAGES.SENT.TAB'/></a></li>
+                            <li id="compose-tab"><a data-toggle="tab" href="#compose"><@spring.message 'MESSAGES.COMPOSE.TAB'/></a></li>
+                        </ul>                        
+                        <div id="inbox" class="tab-pane active">
```

Comment:
sectiunile de inbox si outbox sunt mai mari decat cuprinde macro-ul actual (intreg codul care incepe cu <div id="inbox"/<div id="outbox" e similar si poate face parte din macro). Diferentele dintre cele doua sectiuni (tipul sectiunii - inbox sau outbox, faptul ca una e activa din start si cealalta nu) pot fi transmise in macro ca parametri (type si isActive). Oriunde apare acum termenul de inbox sau outbox o sa fie inlocuit in macro cu valoarea parametrului type. De asemenea, lista de mesaje trebuie transmisa ca parametru, pentru ca fiecare sectiune va afisa liste diferite de mesaje. Redenumeste macro-ul din createMail in createMessagesSection si variabila receivedMessage in message, deoarece in interiorul macro-ului lucrezi pe un caz general, nu pe un caz  specific.